zhaquirks/philips/__init__.py

```diff
@@ -218,3 +218,24 @@ def send_press_event(click_count):
         else:
             action = f"{button}_{press_type}"
             self.listener_event(ZHA_SEND_EVENT, action, event_args)
+
+
+class PhilipsEffectCluster(CustomCluster):
+    """Philips effect cluster."""
+
+    cluster_id = 0xFC03
+    ep_attribute = "philips_effect"
+
+    server_commands = {
```

Comment:
Ideally, we should also use the new "zigpy command definition style", similar to this:
https://github.com/zigpy/zigpy/blob/1853ce109de5d1528333e9c78c325131f8aeed3a/zigpy/zcl/clusters/general.py#L570-L573